src/ios/TLRKFeedback.m

```diff
@@ -34,6 +39,36 @@ -(void)initialize: (CDVInvokedUrlCommand *)command
         uid = objc_msgSend(objc_msgSend(UIDevice.currentDevice, identifierForVendorSEL), @selector(UUIDString));
     }
 
+	// iOS localization specific options
+    // TODO: refactor the code for DRY issues
+    // using UTF8 strings to allow displaying Chinese, Cyrillic, etc. chars 
+    // using NSNull comparison for JSON values set to null
+	if(command.arguments.count == 3 && command.arguments[2]) {
+		NSDictionary *feedbackOptions = command.arguments[2];
+
+		NSDictionary *iOSLocalizationOptions = [feedbackOptions valueForKey:iOSFeedbackOptionsKeyName];
+		if(iOSLocalizationOptions != nil ) {
+			NSString* feedbackSentAlertTitle = [iOSLocalizationOptions valueForKey:feedbackSentAlertTitleSettingName];
+			if (feedbackSentAlertTitle != nil && ![feedbackSentAlertTitle isKindOfClass:[NSNull class]]) {	
+				const char *cString = [feedbackSentAlertTitle UTF8String];
+				[TKFeedback feedbackSettings].feedbackSentAlertTitle = [[NSString alloc] initWithUTF8String:cString];
+		    }
+
+			NSString* feedbackSentAlertText = [iOSLocalizationOptions valueForKey:feedbackSentAlertTextSettingName];
```

Comment:
It is widespread to declare pointers as <type> *<name>. You can also refer to this [style guide](https://github.com/raywenderlich/objective-c-style-guide) for Objective-C for more ideas.